src/open_samus_returns_rando/specific_patches/chozo_seal_patches.py

```diff
@@ -45,5 +47,76 @@ def patch_dna_check(editor: PatcherEditor):
     editor.replace_asset(file_name, systemmechdna)
 
 
+class NewChozoSeal(typing.NamedTuple):
+    scenario: str
+    ap_coordinates: tuple[(float, float, float)]
+    platform_coordinates: tuple[(float, float, float)]
```

Comment:
Oops, that was wrong on my side! We are actually just using a `list[float]` for both.